src/components/Input/Input.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+import styles from "./Input.module.scss";
+
+var classNames = require("classnames");
+
+export type InputProps = Omit<
+  React.InputHTMLAttributes<HTMLInputElement>,
+  "onChange" | "value"
+> & {
+  /** Значение поля */
+  value?: string;
+  /** Callback, вызываемый при вводе данных в поле */
+  onChange?: (value: string) => void;
+};
+
+const Input: React.FC<InputProps> = ({
+  value,
+  onChange,
+  ...props
+}: InputProps) => {
+  return (
+    <input
+      {...props}
+      type="text"
+      className={classNames(
+        styles.input,
+        { input_disabled: props.disabled },
```

Comment:
мне кажется, это не то, что ожидается. при props.disabled === true у тебя применится глобальный класс input_disabled, но у нас он нигде не определён – определён класс в модуле. поэтому нужно написать так:
```suggestion
        { [styles.input_disabled]: props.disabled },
```
прошу поправить здесь и где ещё встретишь

а ещё – просто на заметку – можно писать так и иногда это будет короче и красивее:
```suggestion
        props.disabled && styles.input_disabled,
```